### PR TITLE
Make the decision about whether to copy on upsert more flexible

### PR DESCRIPTION
**Pull request by rjmac**
Also configure it in a less-silly way.

### CODE REVIEW

**Comment by rjmac** [comment]
..huh, that wasn't a build failure I expected.

**Comment by rjmac** [comment]
Oh, that's a silly mistake in a thing I adjusted at the last minute, of course.

File: store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala [hunk 1]
```diff
@@ -53,14 +53,8 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
...
-  val rowsChangedPreviewConfig = curator match {
-    case Some(c) =>
-      val config = new RowsChangedPreviewConfig.ZKClojure(c, "/soql-postgres-secondary")
-      config.start()
-      config
-    case None =>
-      RowsChangedPreviewConfig.Default
-  }
+  val rowsChangedPreviewConfig =
+    new RowsChangedPreviewConfig.Clojure(storeConfig.transparentCopyFunction.getOrElse(RowsChangedPreviewConfig.default))
```
Comment (vickingan) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala:57]: Not a clojure expert, but `transparentCopyFunction` and `default` look like same function?

Comment (rjmac) [reply store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala:57]: They do and are!  But we can change transparentCopyFunction in apps_marathon.

**Review by vickingan — APPROVED** [review]
👍 LGTM

**Pull request merged**